client/package.json

```diff
@@ -13,6 +13,7 @@
     "redux": "^4.0.4",
     "redux-form": "^8.2.6",
     "redux-thunk": "^2.3.0",
+    "styled-components": "^4.4.1"
```

Comment:
This actually breaks the code, since it's missing a comma... 
How did you add this line here? Was this part of your rebase?

Reply:
Oh i really quickly changed it on github, I meant to delete the yarn thing